### ISSUE TITLE
about tpuart2

### ISSUE THREAD

**Issue by leoujz** [knxd/knxd#298]
I have found that knxd supports tpuart, but I don't know tpuart2 is supported or not. Thanks very much!

**Comment by smurfix** [comment]
Yes it is. It's autodetected and works a lot better than the original tpuart.

**Comment by leoujz** [comment]
I met a problem which I thought was caused by tpuart2. I'm using siemens tpuart2 device, through which I can recieve correct packets from knx bus, with "knxtool vbusmonitor1 ip:localhost". But when I send any data, the siemens knx device responds me with 0x47, I have googled that 0x47 might be a "received data error" message. 
When I want to send a packet to knx bus through siemens knx device, such as "knxtool on ip:localhost 0/0/1", under knxd version 0.14.18, the knxd service just die with message "Driver timed out trying to send"; under knxd version 0.11.18 and version 0.14.17,  the knxd service running well but no data received on the knx bus.
I also tried send data to the serial port directly by python, the siemens knx device always repsonds me with 0x47.
